language/sygus.tex

```diff
@@ -2196,7 +2256,7 @@ \subsection{Satisfying Semantic Specifications}
 whose grammars are $G_1, \ldots, G_n$.
 Let $\Psi_{\alpha}$ be the formula:
 \[
-\bigwedge_{i=1}^{n} \forall X_i f( X_i ) \teq t_i
+\bigwedge_{i=1}^{n} \forall X_i. f( X_i ) \teq t_i
```

Comment:
[minor]

`ldotp` in place of `.` renders spaces correctly.
(https://tex.stackexchange.com/a/393087)